python/rikai/torch/data.py

```diff
@@ -40,10 +41,12 @@ class Dataset(IterableDataset):
 
     Parameters
     ----------
-    uri : str
+    uri_or_df : str, Path, pyspark.sql.DataFrame
```

Comment:
do we want to be consistent between `uri_or_df` and `dataset_ref`?
I like `dataset_ref` more , it can be used for "query" as well.

Reply:
I'm going to rename it `data_ref` throughout (i like dataset_ref better as well but it's weird to pass `dataset_ref` to the `Dataset` constructor. I don't want the users to get confused thinking `dataset_ref` is a reference to another instance of `rikai.torch.data.Dataset`.